### PR TITLE
Submitters are advised to submit “Device, Applied not done” instead o…

### PR DESCRIPTION
**Pull request by dczulada**
…f “Device, Order not done” for the VTE-1, VTE-2, and VTE-6 eCQM measures.

### CODE REVIEW

File: templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.9.cat1.erb [hunk 1]
```diff
@@ -30,23 +30,20 @@
...
-  <act classCode="ACT" moodCode="RQO" <%== negation_indicator(entry) %>>
+  <!--  Submitters are advised to submit “Device, Applied not done” instead of “Device, Order not done” for the VTE-1, VTE-2, and VTE-6 eCQM measures. -->
+  <procedure classCode="PROC" moodCode="EVN" <%== negation_indicator(entry) %>>
+    <!-- Procedure Activity Procedure -->
+    <templateId root="2.16.840.1.113883.10.20.22.4.14" extension="2014-06-09"/>
+    <!-- Device Applied -->
+    <templateId root="2.16.840.1.113883.10.20.24.3.7" extension="2014-12-01"/>
     <id root="1.3.6.1.4.1.115" extension="<%= entry.id %>"/>
-    <code code="SPLY" codeSystem="2.16.840.1.113883.5.6"/>
+    <code code="360030002" codeSystem="2.16.840.1.113883.6.96" codeSystemName="SNOMED CT" displayName="application of device"/>
```
Comment (okeefm) [inline templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.9.cat1.erb:40]: QRDA I R3 IG has the Code System Name as `SNOMED-CT` (with the dash). Not sure which is correct, or if it makes a difference.

Comment (okeefm) [reply templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.9.cat1.erb:40]: Though when the give the actual codes for the Device Applied, they omit the dash.

File: templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.9.cat1.erb [hunk 2]
```diff
@@ -30,23 +30,20 @@
     <%== render(:partial => 'reason', :locals => {:entry => entry, :reason_oids=>field_oids["REASON"]}) %>
   </supply>
   <% else %>
-  <act classCode="ACT" moodCode="RQO" <%== negation_indicator(entry) %>>
+  <!--  Submitters are advised to submit “Device, Applied not done” instead of “Device, Order not done” for the VTE-1, VTE-2, and VTE-6 eCQM measures. -->
+  <procedure classCode="PROC" moodCode="EVN" <%== negation_indicator(entry) %>>
```
Comment (okeefm) [inline templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.9.cat1.erb]: Generic question: should we be associating a code with this template? As there is currently no place to add a code other than the hardcoded `application of device` SNOMED code.

**Comment by mayerm94** [comment]
Should these changes also be applied to templates/cat1/r3_1/_2.16.840.1.113883.10.20.24.3.9.cat1.erb ?

**Comment by okeefm** [comment]
According to @dczulada, "Cat I R3.1 was updated to include the “wrappers” that allow for negation".  I believe that means we don't need these changes in R3.1.

**Review by edeyoung — COMMENTED** [review]
I've asked Matt to review.

File: templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.9.cat1.erb (hunk 1 above)
Comment (okeefm) [reply templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.9.cat1.erb:40]: HDS has, in various places, `SNOMED CT`, `SNOMED-CT`, and `SNOMEDCT`. I'm going to say this doesn't matter at this juncture, but we should standardize at some point.

**Review by okeefm — APPROVED** [review]

**Review by edeyoung — APPROVED** [review]
Matt have reviewed and approves the changes.

**Pull request merged**